disperser/apiserver/server_v2.go

```diff
@@ -199,3 +203,75 @@ func (s *DispersalServerV2) RefreshOnchainState(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (s *DispersalServerV2) GetPaymentState(ctx context.Context, req *pb.GetPaymentStateRequest) (*pb.GetPaymentStateReply, error) {
+	// validate the signature
+	if err := s.authenticator.AuthenticatePaymentStateRequest(req); err != nil {
+		return nil, api.NewErrorInvalidArg(fmt.Sprintf("authentication failed: %s", err.Error()))
+	}
+	// on-chain global payment parameters
+	globalSymbolsPerSecond := s.meterer.ChainPaymentState.GetGlobalSymbolsPerSecond()
+	minNumSymbols := s.meterer.ChainPaymentState.GetMinNumSymbols()
+	pricePerSymbol := s.meterer.ChainPaymentState.GetPricePerSymbol()
+	reservationWindow := s.meterer.ChainPaymentState.GetReservationWindow()
```

Comment:
Right now, these are configured as parameters in the api server. Should they live onchain instead?

Reply:
yes, this would be implemented in PR https://github.com/Layr-Labs/eigenda/pull/835